cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,76 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+// const faker = require('faker');
 
 describe('Settings page', () => {
-  before(() => {
-
-  });
-
-  beforeEach(() => {
-
-  });
-
   it('should provide an ability to update username', () => {
-
+    cy.createUser().then(() => {
+      const user = Cypress.env('user');
```

Comment:
Verify that `Cypress.env('user')` is correctly set up to retrieve the user details. This is crucial for the login process to work correctly.